packages/mlgmpidl/mlgmpidl.1.2.15-1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+authors: ["Bertrand Jeannet" "Nicolas Berthier"]
+maintainer: "Nicolas Berthier <m@nberth.space>"
+dev-repo: "git+https://github.com/nberth/mlgmpidl.git"
+bug-reports: "https://github.com/nberth/mlgmpidl/issues"
+homepage: "https://www.inrialpes.fr/pop-art/people/bjeannet/mlxxxidl-forge/mlgmpidl/"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+build: [
+ [
+   "sed"
+   "-ib"
+   "-e"
+   "s/checkprefix \"\\$cc \\$cflags\" mpfr\\.h/checkprefix \"\\$cc \\$cflags -I\\$gmp_prefix\\/include\" mpfr.h/g"
+   "configure"] {os = "macos" & os-distribution = "homebrew"}
+  ["sh" "-exec" "./configure --absolute-dylibs --mpfr-prefix ${HOMEBREW_PREFIX:-/opt/homebrew}/opt/mpfr --gmp-prefix ${HOMEBREW_PREFIX:-/opt/homebrew}/opt/gmp"]  { os = "macos" & os-distribution = "homebrew" }
+  ["sh" "./configure" "--absolute-dylibs" {os = "macos"} ]  { !(os = "macos"& os-distribution = "homebrew") }
+  [make]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "3.12.1"}
+  "ocamlfind" {build & >= "1.5.6"}
+  "camlidl" {!= "1.10"}
+  "conf-gmp"
+  "conf-mpfr"
+  "conf-perl" {build}
+  "bigarray-compat"
+]
+conflicts: [
+  "mlgmp"
+  "apron" {= "20140725"}
+  "apron" {= "20150518"}
+]
+synopsis: "OCaml interface to the GMP library"
+url {
+  src: "https://github.com/nberth/mlgmpidl/archive/1.2.15.tar.gz"
+  checksum: "sha512=bffa295d7eac0a35ba95b8fefa46f5db02efa1f58e4d6b4da73901de74535523ad263354f2672318361ed0f76029df1ef85f745ce245dc73dddffb9d328dc7a8"
```

Comment:
```suggestion
  checksum: [
      "sha512=bffa295d7eac0a35ba95b8fefa46f5db02efa1f58e4d6b4da73901de74535523ad263354f2672318361ed0f76029df1ef85f745ce245dc73dddffb9d328dc7a8"
    "md5=fba69df33295a427e093cad67114a5d6"
  ]
```

even though md5 is weak, keeping it around makes it clearer that the tarball is expected to be the same as in the 1.2.15 version of the package: the diff keeps the hash, just adds another one